contract/bridge/contracts/VaultReward.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
+import "./IVaultRegistry.sol";
+
+contract VaultReward is Initializable {
+  using SafeMathUpgradeable for uint256;
+
+  struct LockedVault {
+    uint256 lockStartAt;
+    uint256 lockPeriod;
+    uint256 lockExpireAt;
+    uint256 rewardClaimAt;
+    uint256 collateralUpdatedAt;
+    uint256 accClaimableRewards;
```

Comment:
Working on #93. Most likely insert list of stakers in here later.

Reply:
Yep. you're right. Working on it.